claims_db/db_loader/scripts_general/copy_into.R

```diff
@@ -160,7 +183,15 @@ copy_into_f <- function(
   }
   
   ### Create table if it doesn't exist
-  if (DBI::dbExistsTable(conn, DBI::Id(schema = to_schema, table = to_table)) == F) {
+  if (DBI::dbExistsTable(conn, DBI::Id(schema = to_schema, table = to_table)) == F & batch_id_assign == T) {
+    DBI::dbExecute(conn, glue::glue_sql(
+      "CREATE TABLE {`to_schema`}.{`to_table`} (
+          {DBI::SQL(glue_collapse(glue_sql('{`c(names(table_config$vars),batch_id_var_name)`} {DBI::SQL(c(table_config$vars,batch_id_var_type))}',
```

Comment:
Apparently there is a glue__sql_collapse version now that returns a SQL object, which could simplify this code a bit: https://glue.tidyverse.org/reference/glue_collapse.html